Code from To Do List project

Yazino12 opened the pull request:
### In this pull:

- I copied all files and folders from the `To-Do List` project

**Changes made:** 

- Use semantic tags to wrap elements
- Remove unwanted styles in `style.css`
- Modify how to store data in local storage

Yazino12 committed:
Copy your code from the To Do list project

Benmuiruri commented on dist/index.html:
```diff
@@ -0,0 +1,41 @@
+    <title>To-do list</title>
+  <script defer src="/todo-list/index.bundle.js"></script></head>
+  <body>
+    <div class="list-wrapper">
```
- [x] You should consider using a more semantic tag to wrap all the elements in the webpage. A tag like `main` instead of `div`

Yazino12 replied:
Thanks, @Benmuiruri for the review. I'll implement these changes right away ✔.

Benmuiruri commented on src/style.css:
```diff
@@ -0,0 +1,181 @@
+/* ADD NOTES SECTION  */
+.add {
+  display: flex;
+  justify-content: space-between;
```
You do not need the `justify-content: space-between `because you have `flex: 1` on the `div` holding the `input` and a specified width for the add button. Therefore, I think by default your `input` will take up the remaining space not occupied by the add button

Benmuiruri commented on src/index.js:
```diff
@@ -0,0 +1,45 @@
+    this.taskArray = taskArray;
+  }
+
+  render = () => {
```
You have a tiny bug in how you load tasks from `local storage`. 

- When you mark something as completed its completed status is correctly updated to `true`. 
- However, when you delete a task, whether it is completed or not, when the task list is updated, all existing tasks that were marked as completed are all marked as incomplete and their completed status are updated to `false` for all tasks. 
-

Benmuiruri reviewed (changes requested):
## Hi @Yazino12 

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . Great job on following Javascript best practices. You followed all _DRY_, _KISS_, and _YAGNI_ rules.
# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me if something is not 100% clear.

Yazino12 committed:
Apply changes requested
Use semantic tags to wrap elements
Remove unwanted styles in style.css
Modify how to store data to local storage

Benmuiruri reviewed (approved):
Hi @Yazino12 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![clap](https://media0.giphy.com/media/j2GPljpvL8Lulm3d0X/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_Keep up the good work_
